rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -850,46 +867,21 @@ URI getParentId() {
         return parentId;
     }
 
-    public boolean afterLRANotification() {
+    public boolean endLRANotification() {
         boolean notifiedAll = true;
 
-        if (afterLRAListeners != null) {
+        if (endLRAListeners != null) {
             Client client = ClientBuilder.newClient();
 
             try {
-                Iterator<AfterLRAListener> listeners = afterLRAListeners.iterator();
+                Iterator<EndLRAListener> listeners = endLRAListeners.iterator();
 
                 while (listeners.hasNext()) {
-                    AfterLRAListener participant = listeners.next();
-                    Response response = null;
-
-                    try {
-                        Future<Response> asyncResponse = client.target(participant.notificationURI)
-                                .request()
-                                .header(LRA.LRA_HTTP_ENDED_CONTEXT_HEADER, id)
-                                .header(LRA_HTTP_RECOVERY_HEADER, participant.recoveryId.toASCIIString())
-                                .async()
-                                .put(Entity.text(status.name()));
-
-                        response = asyncResponse.get(PARTICIPANT_TIMEOUT, TimeUnit.SECONDS);
-
-                        if (response.getStatus() == 200) {
-                            listeners.remove();
-                        } else {
-                            notifiedAll = false;
-                        }
-
-                        response = null;
-                    } catch (WebApplicationException | InterruptedException | ExecutionException | TimeoutException e) {
+                    EndLRAListener participant = listeners.next();
+                    if (participant.call(client)) {
+                        listeners.remove();
+                    } else {
                         notifiedAll = false;
-
-                        if (LRALogger.logger.isInfoEnabled()) {
-                            LRALogger.logger.infof("Could not notify AfterLRA listener at %s (%s)", participant.notificationURI, e.getMessage());
```

Comment:
I wonder why this logging was removed?

Reply:
This code was just refactored to the `call` method in AfterLRAListener - https://github.com/jbosstm/narayana/pull/1541/files/9380ad2ffdfb0716aae6ab743a4b6dca81a3d97d#diff-57ec69ca0d35bbed0ad018339b532a7dR930

Reply:
Sorry, I see that now thanks!